tsconfig.json

```diff
@@ -7,6 +7,8 @@
     "noImplicitAny": false,
     "resolveJsonModule": true,
     "rootDir": ".",
+    // TODO: Marp team does not allow to enable "skipLibCheck". Please revert when improved MathJax typing.
```

Comment:
I hope so. However, It perhaps will be unsolved forever. https://github.com/mathjax/MathJax-src/issues/169
If marp team want to avoid it, we should open new issue with enough reason.

Reply:
In https://github.com/marp-team/marp-react/pull/22#discussion_r372460209 we could fix upstream and there was a workaround (downgrade), but in this case `skipLibCheck` is just a only workaround. To get MathJax integration for now, we have no other choices but to enable.

There is no member of Marp team to interest in math libs, so would difficult to contribute into upstream.